server/Dockerfile

```diff
@@ -6,13 +6,14 @@ VOLUME /timely-server-src
 VOLUME /timely
 
 RUN yum -y update && \
-  yum -y install java-1.8.0-openjdk-devel maven openssl apr && \
-  mkdir -p /timely/bin
+  yum -y install java-1.8.0-openjdk-devel maven openssl apr wget && \
+  mkdir -p /timely/bin && \
+  wget -O /usr/lib/jvm/jre/lib/ext/bcprov-jdk15on-155.jar https://www.bouncycastle.org/download/bcprov-jdk15on-155.jar && \
```

Comment:
Why not just add EPEL repo and do `yum install bouncycastle`?

Also, the standard location for jars is in `/usr/share/java`, with symlinks pointing to this location if necessary. That way, the `/usr/bin/build-classpath` tool works properly.